### PR TITLE
Add default response

### PR DESCRIPTION
**Pull request by KichiginaTatiana**
https://github.com/mattherman/MbDotNet/issues/64

### CODE REVIEW

File: MbDotNet/Models/Imposters/HttpImposter.cs [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
     {
         [JsonProperty("stubs")]
         public ICollection<HttpStub> Stubs { get; private set; }
 
-        public HttpImposter(int? port, string name, bool recordRequests = false) : base(port, Enums.Protocol.Http, name, recordRequests)
+        /// <summary>
+        /// Optional default response that imposter sends back if no predicate matches a request
+        /// </summary>
+        [JsonProperty("defaultResponse", NullValueHandling = NullValueHandling.Ignore)]
+        public HttpResponseFields DefaultResponse { get; set; }
```
Comment (mattherman) [inline MbDotNet/Models/Imposters/HttpImposter.cs]: Please make this a `private set` on each imposter type to be consistent with the other properties.

File: mb.pid [hunk 2]
```diff
@@ -0,0 +1 @@
+85932
```
Comment (mattherman) [inline mb.pid]: Remove this file, and if you want you could add `mb.pid` and `mb.log` to the `.gitignore`.

**Review by mattherman — CHANGES_REQUESTED** [review]
Just a couple small changes + the merge conflicts, but then it should be good to go

File: MbDotNet/Models/Imposters/RetrievedImposter.cs [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
     /// <summary>
     /// The base class for a retrieved imposter.
     /// </summary>
-    /// <typeparam name="T">The request type this imposter contains</typeparam>
+    /// <typeparam name="TRequest">The request type this imposter contains</typeparam>
+    /// <typeparam name="TResponseFields">The response fields type this imposter contains</typeparam>
     /// <typeparam name="TDefaultResponse">The request type this imposter contains</typeparam>
-    public abstract class RetrievedImposter<T, TDefaultResponse>
-        where T : Request
+    public abstract class RetrievedImposter<TRequest, TResponseFields, TDefaultResponse>
```
Comment (mattherman) [inline MbDotNet/Models/Imposters/RetrievedImposter.cs]: Is there ever a situation where `TResponseFields` will be different from `TDefaultResponse`? I think we can support this feature without adding an additional generic type parameter.

**Review by mattherman — APPROVED** [review]

**Comment by mattherman** [comment]
I've published new packages that include all four of your changes:
* https://www.nuget.org/packages/MbDotNet/4.3.3
* https://www.nuget.org/packages/MbDotNet/5.0.0-rc6

They should be available soon. Thanks again!

**Comment by KichiginaTatiana** [comment]
> I've published new packages that include all four of your changes:
> 
> * https://www.nuget.org/packages/MbDotNet/4.3.3
> * https://www.nuget.org/packages/MbDotNet/5.0.0-rc6
> 
> They should be available soon. Thanks again!

thanks!

**Pull request merged**